src/SprykerEco/Yves/Heidelpay/Form/AbstractHeidelpaySubForm.php

```diff
@@ -63,4 +64,12 @@ public function getTemplatePath(): string
     {
         return static::PAYMENT_PROVIDER . '/' . static::PAYMENT_METHOD_TEMPLATE_PATH;
     }
+
+    /**
+     * @return string
+     */
+    public function getProviderName()
```

Comment:
Are we using strict (return)typehinting yet in eco? Then you can use `: string` here

Reply:
Done, thanks for the hint.

Reply:
Ok